guides/php_upgrade.md

```diff
@@ -33,33 +34,36 @@ composer self-update --2
 
 :::: tabs
 ::: tab "NGINX"
-After upgrading to PHP 8.0, you will most likely need to update your NGINX configuration. Your configuration file
+After upgrading to PHP 8.1, you will most likely need to update your NGINX configuration. Your configuration file
 is most likely called `pterodactyl.conf` and located in the `/etc/nginx/sites-available/` directory, or if on CentOS,
 `/etc/nginx/conf.d/`.
 
 Make sure to update the path in the command below to reflect the actual location of your configuration file.
+
 ``` bash
-sed -i -e 's/php7.[0-4]-fpm.sock/php8.0-fpm.sock/' /etc/nginx/sites-available/pterodactyl.conf
+sed -i -e 's/php7.[0-4]-fpm.sock/php8.1-fpm.sock/' /etc/nginx/sites-available/pterodactyl.conf
```

Comment:
Shouldn't this look for `php[7|8].[\d]-fpm.sock`? Seems like we could match even existing 8.1 installs at this point, not a big deal.

Reply:
You're right, I missed this.  `\d` doesn't exist in sed for regular expression as it's a switch. We can use `0-9` or `\w.`. The first is probably more foolproof as it matches digits only?